server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -1100,12 +1116,33 @@ export function configureWebSocketServices(
 							: [contentBatch];
 
 						for (const content of contents) {
-							const signalMessage: ISignalMessage = {
-								clientId,
-								content,
-							};
+							let signalMessage: ISignalMessage;
+							let roomId: string;
+							const feature_submit_signals_v2 = "submit_signals_v2";
+
+							if (
+								supportedFeaturesMap.get(clientId)?.clientSupportedFeatures?.[
+									feature_submit_signals_v2
+								]
+							) {
```

Comment:
Does a client need to say they support that feature? Or can they just send?

Reply:
On initial the connection message, the client specifies that they support the submit_signals_v2 feature. They can't send targeted signals without this.

Reply:
see: https://github.com/microsoft/FluidFramework/blob/22b05b09c0881a04633fae4712c794da77909411/packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts#L301C3-L303C5